modules/KoreBuild.Tasks/KoreBuild.Tasks.csproj

```diff
@@ -10,6 +10,7 @@
     <Content Include="*.targets" CopyToPublishDirectory="PreserveNewest" />
     <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\Utilities\*.cs" />
     <Compile Include="..\..\shared\Utilities\MSBuildListSplitter.cs" />
+    <Compile Include="..\BuildTools.Tasks\Utilities\FileHelpers.cs" Link="Utilities\FileHelpers.cs" />
```

Comment:
Curious what that link property is and why this one has it while others don't.

Reply:
It's a UI thing. It tells VS where to display this in the Solution Explorer